mw-webapp/src/component/input/Input.tsx

```diff
@@ -69,13 +79,14 @@ export const Input = (props: InputProps) => {
    * Event handler for the input change event
    */
   const onChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-    props.onChange(event.target.value);
+    props.onChange(props.type === "number" ? event.target.value.slice(0, MAX_AMOUNT_CHARACTERS) : event.target.value);
```

Comment:
why? does max attribute does not work?

Reply:
Max works in arrows inside input? but you alsp can write any number manually so for dont allow it I added this logic